QUARKUS-2391.md

```diff
@@ -0,0 +1,38 @@
+# QUARKUS-2391 Promote the OpenTelemetry extension from TP to full support
+JIRA:https://issues.redhat.com/browse/QUARKUS-2391
+## Scope of the testing
+Quarkus already supported Opentelemetry as a tech preview, but now we move towards full support, then we will extend our test coverage.
+### Existing tests
+In the current quarkus-test-suite exist two modules dedicated to opentelemetry in the monitoring folder, they are:
+
+- opentelemetry [monitoring-opentelemetry]
+- opentelemetry-reactive [monitoring-opentelemetry-reactive]
+
+These tests utilize trace export to Jaeger (all-in-one pod) components and context propagation through a Ping-Pong application with 3 pods (ping service,pong service, jaeger-all-in-one).
+The traces are directly sent to the Jaeger collector, and they also test Quarkus gRPC communication between services and SSE (Server-Sent Events).
+
+On the other hand, also other modules use ```quarkus-opentelemetry``` dependency:
+- javaee-like-getting-started
+- build-time-analytics
+- http/graphql-telemetry
+- http/vertx-web-client
+- sql-db/narayana-transactions (where is tested JDBC Opentelemetry instrumentation)
+
+#### Test scenarios
```

Comment:
Are these scenarios already exist or do we need to add them?

Reply:
No, they are not, we need to add them.